style.css

```diff
@@ -12957,6 +12877,80 @@ ul.download-versions {
   margin-bottom: 0;
 }
 
+.nhsuk-bordered-tabs-container {
+  padding-top: 8px;
+  background-color: #ffffff;
+  border-bottom: 2px solid #aeb7bd;
+}
+
+.nhsuk-bordered-tabs-container .nhsuk-bordered-tabs {
+  padding: 0;
+  margin-bottom: 0;
+  display: flex;
+}
+
+.nhsuk-bordered-tabs-container .nhsuk-bordered-tabs .nhsuk-bordered-tabs-item {
+  display: inline-block;
+  padding: 16px 24px;
+  margin-bottom: 0;
+  font-size: 90%;
+}
+
+.nhsuk-bordered-tabs-container .nhsuk-bordered-tabs .nhsuk-bordered-tabs-item a {
+  text-decoration: none;
+  word-break: initial;
+}
+
+.nhsuk-bordered-tabs-container .nhsuk-bordered-tabs .nhsuk-bordered-tabs-item a:visited {
+  text-decoration: none;
+  color: #330072;
+}
+
+.nhsuk-bordered-tabs-container .nhsuk-bordered-tabs .nhsuk-bordered-tabs-item a:hover {
+  text-decoration: underline;
+}
+
+.nhsuk-bordered-tabs-container .nhsuk-bordered-tabs .nhsuk-bordered-tabs-item.nhsuk-bordered-tabs-item-active {
+  background-color: #f0f4f5;
+  border: 2px solid #aeb7bd;
+  border-radius: 4px 4px 0px 0px;
+  margin-bottom: -2px !important;
+  border-bottom: 2px solid #f0f4f5;
```

Comment:
Same Here, shall we consider $color_nhsuk- use ?

Reply:
This is the output css. It is generated from nhsuk vars.